flake.nix

```diff
@@ -15,6 +15,7 @@
     {
       packages = {
         inherit (pkgs)
+          gccriscv32-esp32c3-elf-bin
```

Comment:
This is missing a `-`, should be `gcc-riscv32-esp32c3-elf-bin`.

Reply:
Indeed!